lib/form_data.js

```diff
@@ -178,9 +177,27 @@ FormData.prototype._multiPartHeader = function(field, value, options) {
     'Content-Type': [].concat(contentType || [])
   };
 
+  // allow custom headers.
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.](https://www.codacy.com/app/form-data/form-data/file/2279758688/issues/source?bid=2805153&fileBranchId=3104102#l185)